packages/base/src/experiment-manager.ts

```diff
@@ -125,10 +125,12 @@ export class ExperimentManager {
     async closeExperiment(experimentUUID: string): Promise<void> {
         const experimentToClose = this.fOpenExperiments.get(experimentUUID);
         if (experimentToClose) {
-            await this.fTspClient.deleteExperiment(experimentUUID);
-            const deletedExperiment = this.removeExperiment(experimentUUID);
-            if (deletedExperiment) {
-                signalManager().emit(Signals.EXPERIMENT_CLOSED, {experiment: deletedExperiment});
+            const deleteResponse = await this.fTspClient.deleteExperiment(experimentUUID);
+            if (deleteResponse.isOk()) {
```

Comment:
What is the expected behavior when it fails? There could be different behaviors depending on the return code.  Maybe this needs to be handled in a different issue about error handling.

Reply:
Let's remove it and take it on when working on error handling